apps/vs-code-designer/src/app/commands/workflows/openOverview.ts

```diff
@@ -51,7 +51,7 @@ export async function openOverview(context: IAzureConnectorsContext, node: vscod
     panelName = `${vscode.workspace.name}-${workflowName}-overview`;
     workflowContent = JSON.parse(readFileSync(workflowFilePath, 'utf8'));
     baseUrl = `http://localhost:${ext.workflowRuntimePort}${managementApiPrefix}`;
-    apiVersion = '2019-10-01-edge-preview';
+    apiVersion = '2018-11-01';
```

Comment:
If this is working fine on Swechha environment, lets not update this until we are really sure this will fix it